Assets/Scripts/FishBehavior.cs

```diff
@@ -0,0 +1,66 @@
+﻿using UnityEngine;
+
+[RequireComponent(typeof(ParticleSystem))]
+public class FishBehavior : MonoBehaviour {
+
+    const float SPEED = 1f;
+    const float DISTANCE_CHECK = 2f;
+
+    [SerializeField]
+    Transform PathParent;
+
+    ParticleSystem particleSys; //maybe use this to randomize stuff over time???
+    ParticleSystemRenderer particleRenderer;
+
+
+    Vector3 desiredPos;
+    Quaternion desiredRot;
+
+    int currNode = 0;
+
+    void Start() {
+        particleSys = GetComponent<ParticleSystem>();
+        particleRenderer = GetComponent<ParticleSystemRenderer>();
+        SetDesiredNode();
+    }
+
+    void Update() {
+
+        //interpolate to position of current node
+        transform.position = Vector3.Lerp(transform.position, desiredPos, Time.deltaTime * SPEED);
```

Comment:
@MatthewHallberg what is lerp? yinz larpin'?

Reply:
App looks bad,

![Screenshot_1623656230](https://user-images.githubusercontent.com/5590748/123365707-a98f8f80-d52b-11eb-913a-35bd0d5f7911.png)